hikari/impl/rest.py

```diff
@@ -1516,6 +1516,56 @@ def _build_message_payload(  # noqa: C901- Function too complex
 
         return body, form_builder
 
+    def _build_voice_message_payload(
+        self,
+        /,
+        *,
+        attachment: typing.Union[files.Resourceish, messages_.Attachment],
+        waveform: str,
+        duration: float,
+        reply: undefined.UndefinedOr[snowflakes.SnowflakeishOr[messages_.PartialMessage]] = undefined.UNDEFINED,
+        mentions_reply: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
+        reply_must_exist: undefined.UndefinedOr[bool] = undefined.UNDEFINED,
+        flags: typing.Union[undefined.UndefinedType, int, messages_.MessageFlag] = undefined.UNDEFINED,
+    ) -> tuple[data_binding.JSONObjectBuilder, data_binding.URLEncodedFormBuilder]:
+        if flags is undefined.UNDEFINED:
+            flags = messages_.MessageFlag.IS_VOICE_MESSAGE
+        else:
+            flags = messages_.MessageFlag(flags) | messages_.MessageFlag.IS_VOICE_MESSAGE
```

Comment:
```suggestion
        if not flags:
            flags = messages_.MessageFlag.IS_VOICE_MESSAGE
        else:
            flags |= messages_.MessageFlag.IS_VOICE_MESSAGE
```

Should work just as well and is a bit nicer to read